src/cmd-build

```diff
@@ -452,18 +459,36 @@ else
                  "--label=io.openshift.build.versions=machine-os=${buildid}"
                 )
     fi
+    
+    last_build_manifest=()
+    rpm-ostree compose container-encapsulate --help > containerEncapsulateHelp
+    if grep -q "--previous-build-manifest" containerEncapsulateHelp; then
```

Comment:
Any reason to write a persistent file here?  How about just
```
if rpm-ostree compose container-encapsulate --help |grep -q -e "--previous-build-manifest"`; then 
```

? Also note my use of `-e` here because grep can get confused when you give it something that starts with `-` sometimes otherwise.

Reply:
agreed